kolibri/plugins/learn/assets/src/views/classes/ClassAssignmentsPage.vue

```diff
@@ -52,6 +53,16 @@
         lessons: state => state.currentClassroom.assignments.lessons,
       }),
     },
+    mounted() {
+      this.pollForUpdates();
+    },
+    methods: {
+      pollForUpdates() {
+        this.$store.dispatch('classAssignments/updateWithChanges').then(() => {
+          setTimeout(this.pollForUpdates, 30000);
```

Comment:
1. should these be reversed so the timeout happens first and the poll doesn't occur immediately on `mounted`?
2. we should probably assign the ID returned from `setTimeout` to a variable in `data`. Then on `beforeDestroy` we can call e.g. `clearTimeout(timeoutId)`

Reply:
Could we file a follow up issue from this to look at optimizing the backend piece of this to reduce server load?

Reply:
https://github.com/learningequality/kolibri/issues/5757